ext/winevt/winevt_utils.cpp

```diff
@@ -596,8 +596,57 @@ get_description(EVT_HANDLE handle, LANGID langID, EVT_HANDLE hRemote)
   return _wcsdup(result.data());
 }
 
+static int ExpandSIDWString(PSID sid, CHAR **out_expanded)
+{
+#define MAX_NAME 256
+  DWORD len = MAX_NAME, err = ERROR_SUCCESS;
+  SID_NAME_USE sid_type = SidTypeUnknown;
+  char account[MAX_NAME];
+  char domain[MAX_NAME];
+  DWORD result_len = 0;
+  CHAR *formatted = NULL;
+  VALUE vformatted;
+
+  if (!LookupAccountSidA(NULL, sid,
+                         account, &len, domain,
+                         &len, &sid_type)) {
+    err = GetLastError();
+    if (err == ERROR_NONE_MAPPED) {
+      goto not_mapped_error;
+    }
+    else {
+      return -2;
+    }
+
+    goto error;
+  }
+
+  result_len = strlen(domain) + 1 + strlen(account) + 1;
+  formatted = (CHAR *)ALLOCV(vformatted, result_len);
```

Comment:
Should we use `RB_ALLOCV` in recent code?

Reply:
Should we use it? From ruby.h, RB_ALLOCV is old format: https://github.com/ruby/ruby/blob/3f40cebabcf93581ebaecbf76ede38ae2ff22a69/include/ruby/internal/memory.h#L399

Reply:
It means that `ALLOCV` is an old name of `RB_ALLOCV`: https://github.com/ruby/ruby/blob/3f40cebabcf93581ebaecbf76ede38ae2ff22a69/template/Doxyfile.tmpl#L295

Reply:
Fixed by RB_ALLOCV